src/components/swap/SwapButtons/SwapButtons.tsx

```diff
@@ -146,6 +146,12 @@ export function SwapButtons({
     )
   }
 
+  const errorBox = (isSpaceAtTop = true) => (
+    <>
+      {isExpertMode && swapErrorMessage && <SwapCallbackError isSpaceAtTop={isSpaceAtTop} error={swapErrorMessage} />}
+    </>
+  )
+
```

Comment:
Its not a good practice to create components inside functional components. I suggest you to create it outside and pass it as props


```
 const ErrorBox = ({isSpaceAtTop, isExpertMode, swapErrorMessage}) => (
    <>
      {isExpertMode && swapErrorMessage && <SwapCallbackError isSpaceAtTop={isSpaceAtTop ?? true} error={swapErrorMessage} />}
    </>
  )
  
  ```

Reply:
Same comment :)